src/main/java/com/mixpanel/android/mpmetrics/ResourceReader.java

```diff
@@ -109,6 +125,16 @@ private static void readClassIds(Class<?> platformIdClass, String namespace, Map
         }
     }
 
+    protected List<String> getLocalClassNames() {
+        ArrayList<String> localClassNames = new ArrayList<>();
+        for (Class c : ResourceReader.class.getDeclaredClasses()) {
+            localClassNames.add(mResourcePackageName + ".R$" + c.getSimpleName().toLowerCase());
```

Comment:
yikes, adding this hidden dependency here where the name of the class needs to match the name of the resource is very dangerous